frontend/src/components/elements/ArrowVegaLiteChart/ArrowVegaLiteChart.tsx

```diff
@@ -317,9 +318,14 @@ export class ArrowVegaLiteChart extends PureComponent<PropsWithHeight, State> {
 
     const el = this.props.element
     const spec = this.generateSpec()
-    const { vgSpec, view, finalize } = await embed(this.element, spec, {
+    const options = {
       defaultStyle: true,
-    })
+      // Adds interpreter support for Vega expressions that is compliant with CSP
+      ast: true,
+      expr: expressionInterpreter,
```

Comment:
nit: Do you know if this is actually required? I think it is used as default if `ast` is set to `true`:

https://github.com/vega/vega-embed/blob/88fb1224cbb2e51644849a20f1e7ba629629bab9/src/embed.ts#L378

But there is most likely no harm in explicitly specifying this.

Reply:
Agreed it looks like its passed as the default, just thought it would be more straightforward to understand if it was explicitly passed to `expr:`